### PR TITLE
fix(content-block-segmented): align react and wc versions

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#5536

### Description
Alignment between the two Content Block Segmented stories in React and WC.

### Changelog

**New**

- added knobs to change number of ListList items and add border knob to all stories

**Removed**

- the video story variation

### CODE REVIEW

**Comment by kennylam** [comment]
@IgnacioBecerra The web components version has a video in the default story but the React version doesn't.

**Comment by ariellalgilmore** [comment]
looks great! just a few small things: 
1. The spacing between the border and the cta looks different in react and web components
2. the copy in web components is not in the story `with link list`
3. should the additional story be called `with link list` (web components) or `with aside elements`

**Comment by IgnacioBecerra** [comment]
@ariellalgilmore fixed those issues!

File: packages/react/src/internal/components/ContentBlock/ContentBlock.js
Comment (ariellalgilmore) [inline packages/react/src/internal/components/ContentBlock/ContentBlock.js:75]: hmm wondering why this is being added back? It causes the title to go off grid with aside elements. Otherwise looks good!!!
Screen Shot 2021-06-02 at 7 20 45 AM

Comment (IgnacioBecerra) [reply packages/react/src/internal/components/ContentBlock/ContentBlock.js:75]: I removed what I added back since it's more noticeable with that code. I've looked a bit more into this, and I think this is affecting all Content Blocks whenever there's the LinkList open. I'll be opening a new issue capturing this layout problem if there's not one already so it can be addressed properly.

Comment (IgnacioBecerra) [reply packages/react/src/internal/components/ContentBlock/ContentBlock.js]: update: after merging latest changes, I was able to remove the left padding from the content and everything renders as expected. No need to create an issue as it's been already fixed.

**Comment by oliviaflory** [comment]
Hey @IgnacioBecerra, looking good!

A few comments about the knobs:

1. Knobs are not the same between Web components and React, I assume from the issue they should be the same.
   - In Web components Default and With link list are missing `Copy` knob
 
2. Can we control the order of the knobs? Some of the knobs don't seem sequential and are confusing. Suggested order:
**Default**
   - Heading
   - Copy
   - CTA style
   - CTA type
   - Container bottom border

**Link list**
   - Heading
   - Copy
   - Link list heading
   - Number of links
   - CTA style
   - CTA type
   - Container bottom border


3. React small breakpoint: content is shifting off of the columns and is pressed to the edge of the browser, mostly in the React With link list variant. But also seeing it happen to the CTAs in the Default variant

| small | medium|
| --- | --- |
| Screen Shot 2021-06-03 at 5 00 41 PM | Screen Shot 2021-06-03 at 5 00 27 PM |

4. Link list alignment medium breakpoint
In Web components link list hangs on the left side in medium breakpoints (left)
In React link list is inset in the medium breakpoints (right)

From the spec it looks like link list should be inset on the medium breakpoints. I know there is another PR adjusting something in Link list as well...lmk if there should be a separate issue.

Screen Shot 2021-06-03 at 3 04 54 PM

5.React  Link list alignment lg, xl, max
List list isn't aligned to the columns in react, it's hanging on the left side. idk if this was originally captured so LMK if that should be a separate issue.
I believe there is _another_ issue to fix the number of columns link list spans...It should be 3 not 4 :/ 

Screen Shot 2021-06-03 at 5 24 20 PM

**Comment by IgnacioBecerra** [comment]
Thanks for the feedback @oliviaflory!

About the copy knob comment, there's an issue in Web Components in which the copy knob for Content Block stories (and some others as well) aren't able to be updated due to the way they are being rendered. We made a decision to icebox this for now, ([here's the Slack thread for more info](https://cognitive-app.slack.com/archives/GBG66T9NK/p1619795769013200?thread_ts=1619723263.090700&cid=GBG66T9NK))

About the Link List components, I think they could be fixed in another issue since they'll be affecting other components, and I think those other issues may be addressing this as well. 

Otherwise, the other comments have been addressed!

**Comment by oliviaflory** [comment]
Notes from Slack convo with Ignacio:
- Fixing missing bottom border in react
- Fixing padding

**Comment by IgnacioBecerra** [comment]
@oliviaflory These have been fixed in the latest commit.

**Review by oliviaflory — APPROVED** [review]
Looks good @IgnacioBecerra thanks for the updates!

Approved the Percy build even though it's not picking up the video in some story, as we discussed over slack we aren't sure why but the storybook looks good 👍

**Comment by jeffchew** [comment]
@IgnacioBecerra Can you check prettier formatting, looks like that's failing CI checks.

@oliviaflory @IgnacioBecerra I believe we have the video story suppressed in percy since it is loaded asynchronously, it sometimes would cause false positive reports.

**Review by jeffchew — APPROVED** [review]
LGTM!

@IgnacioBecerra @oliviaflory not sure if design needs to be re-approved, there weren't any changes that happened that would require another review since it was just fixing lint issues.

**Pull request merged**
